opt/build.sh

```diff
@@ -52,10 +63,13 @@ download_app_repo() {
   rm -rf ${rootfs_overlay}/opt/enclosures
   rm -rf ${rootfs_overlay}/opt/seedsigner_pubkey.gpg
   rm -rf ${rootfs_overlay}/opt/setup.py
+  rm -rf ${rootfs_overlay}/opt/tests
+  rm -rf ${rootfs_overlay}/opt/tools
+  rm -rf ${rootfs_overlay}/opt/pytest.ini
```

Comment:
can also add requirements-raspi.txt to list of files to delete... since flow-tests merged recently.

Reply:
I'll leave this for a followup PR.